src/main/java/manageezpz/logic/commands/EditTaskCommand.java

```diff
@@ -150,9 +202,24 @@ private Task updateDeadline(Deadline currentTask, String desc, String date, Stri
         return updatedDeadlineTask;
     }
 
+    /**
+     * Updates an Event task.
+     *
+     * @param currentTask Current Event task that is to be updated
+     * @param desc New description of the Event Task
+     * @param date New date of the Event Task
+     * @param startTime New start time of the Event Task
+     * @param endTime New end time of the Event Task
+     * @return Updated Event task
+     */
+
     private Task updateEvent(Event currentTask, String desc, String date, String time) throws ParseException {
         Event updatedEventTask = new Event(currentTask);
 
+        if (!ensureFormatCompliance(prefixStatusHash, desc, date, time)) {
+            throw new ParseException(MESSAGE_EDIT_TASK_NO_EMPTY_VALUES);
```

Comment:
Change this to throw new ParseException(String.format(MESSAGE_EDIT_TASK_NO_EMPTY_VALUES, MESSAGE_USAGE));